src/twisted/conch/insults/helper.py

```diff
@@ -456,6 +456,76 @@ def __bytes__(self):
         return b'\n'.join(lines)
 
 
+    def getHost(self):
+        """
+        @see: L{ITransport.getHost}
+        """
+        raise NotImplementedError()
```

Comment:
```suggestion
        raise NotImplementedError('getHost not implemented')
```
When raising a NotImplementedError I think is best to provide the name... in this way, when the error is raise and you don't have the traceback, you can get a clue from the text of the error

```
>>> str(NotImplementedError())
''
>>> str(NotImplementedError('some_method'))
'some_method'
>>> str(NotImplementedError('some_method not implemented'))
'some_method not implemented' 
```

Reply:
OK, I changed things to `raise NotImplementedError("some message")`